weaviate/collections/classes/internal.py

```diff
@@ -531,6 +527,7 @@ def from_input(
     ObjectSingleReturn[Properties, TReferences],
     ObjectSingleReturn[TProperties, References],
     ObjectSingleReturn[TProperties, CrossReferences],
+    None,
```

Comment:
what is this for?

Reply:
The `fetch_object_by_id` query returns `None` if `len(objects.objects) == 0` but this wasn't captured in the type `QuerySingleReturn` so I've added it here for that